packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -75,15 +75,16 @@ export default class SaytDriverPlugin implements Plugin {
   }
 
   /**
-   * Dispatches an event with the response from the sayt data plugin.
+   * Dispatches an event with the response from the sayt data plugin and the associated searchbox ID.
    * Callback for the Sayt data request event listener.
    *
    * @param event Event that contains the Sayt API request payload.
    */
   fetchSaytData(event: CustomEvent): void {
-    this.sendSaytApiRequest(event.detail)
-      .then((data) => {
-        this.core[this.eventsPluginName].dispatchEvent(this.saytResponseEvent, data);
+    const { query, searchbox } = event.detail;
```

Comment:
This was more along what I had in mind as well still need to pass in the config object. 

```suggestion
    const { query, searchbox, ...config } = event.detail;
    this.sendSaytApiRequest(query, config)
```

Reply:
Actually, in talking with @jmbtutor, a better approach would be to not spread `config`. This makes it a little safer and avoids the issue with sending unexpected information to the Sayt API in the future. 

Will also require changing the signature for `sendSaytApiRequest() slightly`.